lms/djangoapps/program_enrollments/api/v1/tests/test_views.py

```diff
@@ -896,6 +920,25 @@ def test_no_student_key(self, bad_records):
         self.assertEqual(422, response.status_code)
         self.assertEqual('invalid enrollment record', response.data)
 
+    def test_extra_column(self):
```

Comment:
can we avoid the use of 'column' on the lms side since we aren't dealing with CSVs and just refer to this as an extra field/attribute?